core/src/main/resources/hudson/model/Computer/sidepanel.jelly

```diff
@@ -33,7 +33,17 @@ THE SOFTWARE.
       <l:task contextMenu="false" href="${rootURL}/computer" icon="icon-up icon-md" title="${%Back to List}"/>
       <l:task contextMenu="false" href="${rootURL}/${it.url}" icon="icon-search icon-md" title="${%Status}"/>
       <l:task href="${rootURL}/${it.url}delete" icon="icon-edit-delete icon-md" permission="${it.DELETE}" title="${%Delete Agent}"/>
-      <l:task href="${rootURL}/${it.url}configure" icon="icon-setting icon-md" permission="${it.EXTENDED_READ}" title="${%Configure}"/>
+
+      <j:choose>
+        <j:when test="${app.hasPermission(it.CONFIGURE)}">
```

Comment:
```suggestion
        <j:when test="${it.hasPermission(it.CONFIGURE)}">
```

Reply:
seems to work though, =/

Reply:
I've committed it but both seem to work in this case from what I can see

Reply:
Don't grant the permission globally, use project-specific matrix auth and grant it on the specific agent only.

Reply:
Hmm ? That’s what’s there now isn’t it?

Reply:
Ah you mean when I was testing before sure